README.md

```diff
@@ -87,6 +87,12 @@ module.exports.webpack = function(config) {
 };
 ```
 
+You can use this functionality to inject external styles or scripts if your
+components need e.g. Bootstrap to work.
+```js
+config.entry.push('../../../injectStyleTag.js');
```

Comment:
This example is a bit confusing, why would be the extra file be outside the project (`../../..`), since the config file is located in the root of your project, right?

Also, since you gave Bootstrap as an example, it might be useful to have the bootstrap file in the example as well. (unless you need many extra files and it looks ugly, I guess...)